import org.springframework.core.NestedRuntimeException;
public class FunctionUse822 {
public void funcUse() {
NestedRuntimeException nestedruntimeexception = new NestedRuntimeException();
nestedruntimeexception.getRootCause();
}
}